plugins/hook-deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+
+# Show us commands, stop on errors, and fail on missing vars
+set -eux
+
+# Remove any leftovers from past builds
+rm -rf -- /deploy
+
+# Start in a fresh directory
+mkdir /deploy
+cd /deploy
+
+curl -L -o repo.zip https://nightly.link/gwt-plugins/gwt-eclipse-plugin/workflows/build/main/repository.zip
+
+cd /nightly
+
+rm -rf ./*
```

Comment:
Instead, we should make a new directory, and then use that to replace the old one (or change a softlink to point at the new one, then delete the old dir), otherwise we get some significant downtime waiting for the ~300mb repo to download and unzip.